product_extended_segmentation/tests/test_avg_costs.py

```diff
@@ -112,3 +112,15 @@ def test_02_compute_price_with_real_bom(self):
             template_ids=[template_id.id], test=True)
         self.assertEqual(str(res),
                          '{{{0}: 75.0}}'.format(str(template_id.id)))
+
+    def test_03_write_real_cost_product_price_using_wizard(self):
+        template_id = self.prod_e_id.product_tmpl_id
+        self.prod_d_id.write({
+            'cost_method': 'real',
+            'standard_price': 35.0
+        })
+        old_price = self.prod_d_id.standard_price
+        self.env['product.template'].compute_price(
```

Comment:
@osvalr at this moment it could be that `compute_price` method is not updating your `standard_price` field
thus your test could keep failing